charon/dump.py

```diff
@@ -64,7 +64,7 @@ def undump(db, filename):
             atts = doc.pop('_attachments', dict())
             db.save(doc)
             count_items += 1
-            for attname, attinfo in atts.items():
+            for attname, attinfo in list(atts.items()):
```

Comment:
Does this line work? I don't think list is not iterable like this...

Reply:
It seems to give the same result.

Reply:
OK thanks!